main.go

```diff
@@ -193,7 +193,19 @@ func tr1d1um(arguments []string) (exitCode int) {
 			return 1
 		}
 
-		addWebhookHandler := ancla.NewAddWebhookHandler(svc, ancla.HandlerConfig{MetricsProvider: metricsRegistry, V: builtValidators})
+		var webhookDisablePartnerIDs bool
+		err = v.UnmarshalKey(webhookConfigKey+".disablePartnerIDs", &webhookDisablePartnerIDs)
```

Comment:
Is there not a `struct` that is the configuration for webhooks?  This would be better as:

```
type WebhookConfig struct {
    // existing fields

    // new field for this PR
    DisablePartnerIDs bool
}

// unmarshal the entire struct.  Notice that the following code won't need to change if you add new configuration fields
var cfg WebhookConfig
if err := v.Unmarshal(&cfg); err != nil {
    // TODO
}

// now you can use cfg.DisablePartnerIDs as normal
```

Reply:
i've applied this suggestion